### DISCUSSION TITLE
Ubuntu updates to new versions

### DISCUSSION THREAD

**Discussion by djmitche** [GothenburgBitFactory/taskwarrior#2385] [General]
I posted https://bugs.launchpad.net/ubuntu/+source/task/+bug/1909879 to the ubuntu bug tracker, and the response has some good information.  It looks like there are some pending tests and a 5-day timer [here](https://tracker.debian.org/pkg/task).  I'll keep an eye on that and identify any issues.

**Comment by tbabej** [comment]
Looks like the effort was successful!

A different user mentioned that some distros are slow to update the package, maybe we could survey the state across various distributions.

**Reply by mrossinek** [reply]
Yes, Arch is also lacking behind: https://archlinux.org/packages/community/x86_64/task/
The package got flagged out-of-date quite a while ago though..

**Comment by Am3ra** [comment]
Checking in from ubuntu, apt is still on 2.5.1 . Any way to help this along? Or is this a personal issue? ahhahaha

**Reply by kwoot** [reply]
Check the ubuntu packages. Upcoming 21.04 has a newer version: https://packages.ubuntu.com/search?suite=default&section=all&arch=any&keywords=taskwarrior&searchon=names
